docker_plugin/docker_client.py

```diff
@@ -33,7 +34,23 @@ def get_client(daemon_client):
     """
 
     try:
-        return Client(**daemon_client)
+        ctx.logger.info("Daemom connection options: {0}.".format(
+            str(daemon_client)))
+        tp = daemon_client.get('transport_protocol', 'tcp')
+        docker_host = daemon_client.get('host_string', 'localhost')
+        docker_port = daemon_client.get('host_port', 2375)
```

Comment:
except that here you are making the plugin harder to maintain because you have multiple places where defaults are declared...